.flake8

```
@@ -1,2 +1,2 @@
 [flake8]
-ignore = E501,E226,E251,E221
+ignore = E501,E226,E251,E221,E501
```

```suggestion
ignore = E501
```

is enough and the lint still passes with these settings (see https://github.com/SovereignCloudStack/gx-credential-generator/pull/84#pullrequestreview-2011643696)